.ci/es-snapshots/Jenkinsfile_build_es

```diff
@@ -157,6 +157,9 @@ def buildArchives(destination) {
       ./gradlew -p distribution/archives assemble --parallel
       mkdir -p ${destination}
       find distribution/archives -type f \\( -name 'elasticsearch-*-*-*-*.tar.gz' -o -name 'elasticsearch-*-*-*-*.zip' \\) -not -path *no-jdk* -exec cp {} ${destination} \\;
+      ./gradlew -p distribution/docker assemble --parallel
+      docker images "docker.elastic.co/elasticsearch/elasticsearch" --format "{{.Tag}}" | xargs -n1 bash -c 'docker save docker.elastic.co/elasticsearch/elasticsearch:${0} | gzip > distribution/docker/elasticsearch-${0}-docker-image.tar.gz'
```

Comment:
This shouldn't be necessary - the archives should already be within `distribution/docker/build/**/*`

Reply:
Specifically `distribution/docker/build/elasticsearch_test.8.0.0-SNAPSHOT.docker.tar` for 8.0.0.

Reply:
Rory confirmed that release manager also uses docker save, I just removed the copy and zipped it to the destination directory.